utils/log_writer.go

```diff
@@ -0,0 +1,22 @@
+package utils
+
+import (
+	"bufio"
+	"io"
+
+	log "github.com/Sirupsen/logrus"
+)
+
+// NewLogWriter pipes stdout/err logs to logrus
+func NewLogWriter(fields log.Fields, level log.Level) io.WriteCloser {
```

Comment:
I didn't actually plan on this being user-configurable. The task always logs to the `logging` config.  Stream maps to a particular LogLevel: `stdout => InfoLevel` and `stderr => DebugLevel`

It can be configured if we want to add that option. I could put it in as part of the `LogConfig` and optionally choose some default values if it is not provided.